### PR TITLE
Issue #23 Our own text generator

### LINKED ISSUE #23: Use an English vocabulary to generate random phrases
Depends on #22.

The `/random-phrase` endpoint currently generates a random phrase in Latin. Like this:

> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quamquam te quidem video minime esse deterritum. Hic nihil fuit, quod quaereremus. Facile est hoc cernere in primis puerorum aetatulis. Quo modo autem philosophus loquitur? Id enim natura desiderat. Duo Reges: constructio interrete.

This is too long for our screen. I think that on the screen of the demo app perhaps max 3 words can fit — more will go over the screen size. And it is also not funny — a dull Lorem Ipsum text everyone has seen hundreds of times.

Can we generate something ourselves?

For example, we could have a list of adjectives:

- funny
- green
- hard
- nerdy
- wild
- …

and a list of nouns:

- programming
- Python
- clouds
- code
- …

and then create phrases from these like:

- green clouds
- nerdy kangaroo
- furry Python

whatever. Any other ideas are welcome.

When done, I also suggest to write a unit test to cover this function. This unit test might check that the output string contains two words separated by a space, for example.

### PR DESCRIPTION
**Pull request by RomanZorkin**
Closes #23 
In this case I have one linter mistake in "MakeWord" class - "random_word" method. Linter suspects random generator of a security breach:

"S311 Standard pseudo-random generators are not suitable for security/cryptographic purposes. "

It seems to me that the "random_word" function has no binding to security.

I also need to supplement the text files with a large number of words

### CODE REVIEW

File: geopatterns_demo/random_phrase.py [hunk 1]
```diff
@@ -1,6 +1,22 @@
...
+    """Class forms a random word from the transmitted file."""
+
+    def __init__(self, text_file):
+        self.text_file = text_file
+
+    def random_word(self):
+        """Find random word in text file."""
+        with open(self.text_file, 'r') as text:
+            lines = text.readlines()
+            return random.choice(lines).rstrip('\n')
```
Comment (RomanZorkin) [inline geopatterns_demo/random_phrase.py]: There is linter problem:

 S311 Standard pseudo-random generators are not suitable for security/cryptographic purposes. [flake8-bandit]
  return random.choice(lines).rstrip('\n')

Comment (anatoly-scherbakov) [reply geopatterns_demo/random_phrase.py]: `bandit` has to complain about this but we're not using this for any cryptographic or security purposes, so I guess `random.choice()` is pretty much okay. We can ignore this.

It is also a good practice to add a comment to explain why the `noqa` was added.

```suggestion
            # random.choice() is considered to be insecure by flake8-bandit, but we're not using this for any kind of cryptographic or security related purpose.
            return random.choice(lines).rstrip('\n')   # noqa: S311
```

File: geopatterns_demo/random_phrase.py [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
 
 
 def make_random_text():
-    """Method create random phrase using loremipsum library."""
-    return loremipsum.generate(1, 'short')
+    """Glues  and return a phrase from random words."""
+    words_list = [MakeWord(word.value).random_word() for word in Thesaurus]
```
Comment (anatoly-scherbakov) [inline geopatterns_demo/random_phrase.py]:
```suggestion
    words_list = [MakeWord(file_path.value).random_word() for file_path in Thesaurus]
```

The variable should reflect its name, so perhaps `file_path` would be a more appropriate name.

File: geopatterns_demo/models.py [hunk 3]
```diff
@@ -35,3 +35,10 @@ class PhraseDescription(BaseModel):
     """
 
     text: str
+
+
+class Thesaurus(Enum):
+    """List of dictionaries with words."""
+
+    ADJECTIVES = 'geopatterns_demo/dictionary/adjectives.txt'
```
Comment (anatoly-scherbakov) [inline geopatterns_demo/models.py]:
```suggestion
    ADJECTIVES = Path(__file__).parent / 'dictionary/adjectives.txt'
```

Check out `Path` class from `pathlib` library. It helps working on file paths A LOT.

**Review by anatoly-scherbakov — CHANGES_REQUESTED** [review]

File: geopatterns_demo/models.py (hunk 3 above)
Comment (RomanZorkin) [reply geopatterns_demo/models.py]: Whether there is a rule for the correct order of classes. Let's say alphabetically?

**Review by anatoly-scherbakov — APPROVED** [review]

**Pull request merged**